hw/_index.md

```diff
@@ -1,35 +0,0 @@
-# Hardware Specifications
```

Comment:
wait but you're still pointing to `hw/` in `_index.md` at the top level

Reply:
This should be fixed now.